lib/mediawiki_api/wikidata/wikidata_client.rb

```diff
@@ -47,6 +47,18 @@ def create_claim(entity_id, snaktype = "value", property_id, value_data)
         action(:wbcreateclaim, token_type: "csrf", entity: entity_id, snaktype: snaktype, property: property_id, value: value_data)
       end
 
+      def get_entities(ids, site_id)
+        action(:wbgetentities, token_type: false, ids: ids, sites: [site_id])
+      end
+
+      def set_qualifier(claim_id, snaktype = "value", property_id, value_data)
+        action(:wbsetqualifier, token_type: "csrf", claim: claim_id, snaktype: snaktype, property: property_id, value: value_data)
+      end
+
+      def set_reference(statement_id, snaks_data)
+        action(:wbsetreference, token_type: "csrf", statement: statement_id, snaks: snaks_data)
```

Comment:
Since the `reference` parameter is not here, you won't be able to use this to edit a reference. This might be intended. I just want to make sure you are aware of this.

Reply:
Yes, at this time, what I want to do is just adding some references (and qualifiers).  It's enough for me for now.
I think it's good to use keywords arguments for other (not-required) parameters of `wbsetreference`, but I'm not sure other users of this library think so.